pkg/sentry/socket/epsocket/stack.go

```diff
@@ -138,3 +140,24 @@ func (s *Stack) TCPSACKEnabled() (bool, error) {
 func (s *Stack) SetTCPSACKEnabled(enabled bool) error {
 	return syserr.TranslateNetstackError(s.Stack.SetTransportProtocolOption(tcp.ProtocolNumber, tcp.SACKEnabled(enabled))).ToError()
 }
+
+// Forwarding implements inet.Stack.Forwarding.
+func (s *Stack) Forwarding(protocol tcpip.NetworkProtocolNumber) bool {
+	switch protocol {
+	case ipv4.ProtocolNumber, ipv6.ProtocolNumber:
+		return s.Stack.Forwarding(protocol)
+	default:
+		panic(fmt.Sprintf("unsupported protocol %v", protocol))
```

Comment:
Is this needed? It seems like kind of an unfriendly interface. Consider returning false or returning an error.